stdlib/buffer.mli

```diff
@@ -177,3 +177,34 @@ val add_seq : t -> char Seq.t -> unit
 val of_seq : char Seq.t -> t
```

Comment:
It would be nice to add `add_uint8`, `add_int8` and `add_uint16_{ne,le,be}`.

Reply:
I don't think we need add_uint*, since all add/set functions consider only the lsb (and so behave the same on signed and unsigned).